src/inline_lib.ml

```diff
@@ -0,0 +1,92 @@
+open Import
+open Jbuild
+open Build.O
+open! No_io
+
+type rule =
+  { exe: Jbuild.Executables.t
+  ; alias_name: string
+  ; alias_action: (unit, Action.t) Build.t
+  ; alias_stamp: Sexp.t
+  ; last_lib: string
```

Comment:
It might make sense to just add this field to the specification of `Jbuild.Executables.t`.